main.py

```diff
@@ -0,0 +1,26 @@
+from random import randint
+import math
+N = input("Enter amount of numbers: ")
+array = list()
+a = input("Enter first number: ")
+b = input("Enter second number: ")
+for i in range (int(N)):
+    array.append(randint(int(a),int(b)))
+print("" + str(array))
+
+results = list()
+
+print ("Найменше спільне кратне для кожної пари: ")
+for i in array:
+    for k in array:
+        if i!=k:
+            def func(i, k):

```

Comment:
1) functions should be defined after imports
2) Function should be named correctly